### PR TITLE
(SERVER-2331) Merge 6.0.1 into master

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(def ps-version "6.0.1-SNAPSHOT")
+(def ps-version "6.0.2-SNAPSHOT")
```
Comment (Magisus) [inline project.clj]: Do we want to update this to `6.1.0-SNAPSHOT`?

File: dev-resources/puppetlabs/services/certificate_authority/certificate_authority_int_test/infracrl_test/master/conf/ssl/ca/infra_serials [hunk 2]
```diff
@@ -1 +1 @@
-23
+
```
Comment (Magisus) [inline dev-resources/puppetlabs/services/certificate_authority/certificate_authority_int_test/infracrl_test/master/conf/ssl/ca/infra_serials]: Why is this empty now?

Comment (justinstoller) [reply dev-resources/puppetlabs/services/certificate_authority/certificate_authority_int_test/infracrl_test/master/conf/ssl/ca/infra_serials]: infra-crl code should populate the infra_serials file using the certnames in the infra_inventory.txt. That this was set to the serial number of the old certname was an oversight in our original testing. Removing this provides better integration testing (and broke when I regenerated the certs).

File: project.clj (hunk 1 above)
Comment (justinstoller) [reply project.clj]: Yeah, I was just thinking about getting the merge up done, but makes sense to do it now.

Comment (justinstoller) [reply project.clj]: updated

**Pull request merged**